ui/src/index.html

```diff
@@ -1,139 +1,191 @@
 <!DOCTYPE html>
 <html>
```

Comment:
Could you add a lang="en" attribute on this tag for accessibility purpose

Reply:
done